packages/service-library/tests/test_redis.py

```diff
@@ -35,7 +36,7 @@
 ]
 
 pytest_simcore_ops_services_selection = [
-    # "redis-commander",
+    "redis-commander",
```

Comment:
I would leave ti commented, since it takes a while for the Redis-commander to start and slows down the test

Reply:
no. this is automatically removed when running in the CI. and locally it is useful. just use ```--keep-docker-up``` and you are done.